content.tex

```diff
@@ -9,9 +9,9 @@ \section{Teams and their On-Sky Nighttime Responsibilities}
 \end{itemize}
 
 The limit of on-site nighttime staff is due to hotel room availability.  It is not anticipated that staff from the following groups will be regularly joining the nighttime observing crew: DevOps staff, software engineering staff, electrical, electronics, mechanical, or optical staff.
-However, it is anticipated that some of these Rubin staff will participate in out of hours support, either or both remotely or in-person for emergencies to get systems up and running.
+However, it is anticipated that some of these Rubin staff will participate in out of hours support, either or both remotely or in-person, to assist in resolving unexpected issues and/or emergencies that jeopardize system performance or safety.
 
-This document will lay out the different out of hours support which will be needed and must be compensated for.
+This document lays out the different types of out of hours support which will be needed and must be compensated for.
 
 \section{Types of Out Of Hours Support}
```

Comment:
One concept that was in Cristian's original document, but not this one, was the concept of being on-call and receiving ~10% of the wage, but then increasing to 100% if someone actually calls them. Was that dropped intentionally?

Reply:
The allowances section was the only place where it was originally discussed and I've kept it there.  If you think it needs to be up front, we can intro it there.  I just didn't want to repeat the calculation as it was provided later in the document.

Reply:
added to intro:
On-call compensation is 10\% times the hourly wage for the duration of an official on-call shift.  This increases to 100\% times the hourly wage if the person is called to work out of hours time.